tests/publiccloud/aistack_rbac_run.pm

```diff
@@ -0,0 +1,39 @@
+use Mojo::Base 'publiccloud::basetest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use publiccloud::utils;
+use version_utils;
+use transactional qw(trup_call);
+
+sub test_flags {
+    return {fatal => 1, publiccloud_multi_module => 1};
+}
+
+sub run {
+    my ($self, $args) = @_;
+
+    my $instance = $args->{my_instance};
+    my $provider = $args->{my_provider};
+
+    # Get Open WebUI tests and admin credentials
+    my $rbac_tests_url = data_url("aistack/open-webui-rbac-tests.tar.gz");
+    my $admin_email = get_var('OPENWEBUI_ADMIN_EMAIL');
+    my $admin_password = get_var('OPENWEBUI_ADMIN_PWD');
+    record_info("Got Open WebUI admin credentials: $admin_email and $admin_password.");
+    assert_script_run("export OPENWEBUI_ADMIN_EMAIL=$admin_email OPENWEBUI_ADMIN_PWD=$admin_password");
```

Comment:
My python tests get the admin credentials from the environment variables OPENWEBUI_ADMIN_EMAIL and OPENWEBUI_ADMIN_PWD.

Reply:
well, then isn't the credential handling in charge to the pytest code itself?
Setting is available here, but used when tar expanded so that code find the settings already exported: I mean, export here not needed and if you try to remove it, a VR should work the same.

Reply:
It makes sense. Let me remove this line and do a VR.

Reply:
wait :D may be it is needed but a check is ok to do

Reply:
I did the VR, but it failed: https://openqa.suse.de/tests/16204465#step/aistack_rbac_run/34. Somehow I'm not able to access these parameters without exporting the variables. I'll revert it back.

Reply:
ok